tasks/kmt.py

```diff
@@ -1742,3 +1743,79 @@ def _color_result(result: tuple[int, int, int]) -> str:
 
     print(tabulate(table, headers=["Package"] + vm_list))
     print("\nLegend: Successes/Failures/Skipped")
+
+
+@task
+def tag_ci_job(ctx: Context):
+    """Add extra tags to the CI job"""
+    tags: dict[str, str] = {}
+
+    # Retrieve tags from environment variables, with a renaming for convenience
+    environment_vars_to_tags = {
+        "ARCH": "arch",
+        "TEST_COMPONENT": "component",
+        "TAG": "platform",
+        "TEST_SET": "test_set",
+        "MICRO_VM_IP": "microvm_ip",
```

Comment:
Why is the `microvm-ip` useful as a tag?

Reply:
I added it so we can count the number of VMs per instance, and also to be able to easily link jobs that are running in the same VM. Not super-useful but assuming the cost of these tags is negligible I thought it was worth it.